File: packages/app-icon/ios/Plugin/Classes/Results/IsAvailableResult.swift
import Foundation
import Capacitor

@objc public class IsAvailableResult: NSObject, Result {
    let available: Bool

    init(available: Bool) {
        self.available = available
    }

    @objc public func toJSObject() -> AnyObject {
        var result = JSObject()
        result["available"] = available
        return result as AnyObject
    }
}

File: packages/light-sensor/android/src/main/java/io/capawesome/capacitorjs/plugins/lightsensor/classes/results/IsAvailableResult.java
package io.capawesome.capacitorjs.plugins.lightsensor.classes.results;

import androidx.annotation.NonNull;
import com.getcapacitor.JSObject;
import io.capawesome.capacitorjs.plugins.lightsensor.classes.interfaces.Result;

public class IsAvailableResult implements Result {

    private final boolean available;

    public IsAvailableResult(boolean available) {
        this.available = available;
    }

    @Override
    @NonNull
    public JSObject toJSObject() {
        JSObject result = new JSObject();
        result.put("available", available);
        return result;
    }
}
